lib/loader/index.js

```diff
@@ -77,6 +113,24 @@ module.exports = {
 
     validateArguments(url, options, callback);
 
+    if (options.namespace) {
+      namespace = namespacer.namespace(options.namespace);
```

Comment:
The reason I put it there as opposed to the top of the function was so that I could include the validation logic within the `validateArguments` function. To me it makes sense to group all that together in the same function, even if it means I get the namespace a little later. If you still disagree and feel like it should be moved to the top of the function, let me know and I'll update it.